### ISSUE TITLE
Courrier doesn't deliver items.

### ISSUE THREAD

**Issue by Anon6669** [ValveSoftware/Dota2-Gameplay#8193]
### Description

If you have a courier and a divine rapier in stash, the courier will fly to you the whole time but doesn't take any items with it

### Screenshots

I have no screenshots but you can easily re-create the bug because it works in all lobbies, including bot matches

**Comment by anotherCoward** [comment]
The courier is unable to carry or pick up the rapier by design. - All other items from the stash will be delivered fine.

The courier shouldn't take off, if only rapiers are in the stash. Besides that, it it works as intended.

**Comment by Anon6669** [comment]
For me it doesn't even take the other items, and it still keeps flying the whole entire time from base to me without anything.

**Comment by anotherCoward** [comment]
Please describe step by step, what you are doing. 

Tested in Demo Mode
- Bought Rapier
- Moved it to stash
- Bought other Items
- Moved them to stash
- Used Default Shortcut F3 to make the Courier bring the Items
- Courier delived all items except Rapier.
- Used Default Shortcut again after Courier got back
  - Courier moved empty towards me and returned to his spot.

**Comment by Anon6669** [comment]
Do you have discord on which I could show you? I'm not the best at explaining, but here's what I did

Buy Rapier 
Press F3 to deliver it, and it doesn't.
Buy any other items and the courier gets stuck by repeatedly bringing the rapiers only but not taking anything from the stash.
(I just had a minor update so I don't know if it will still work, but I assume it will still be glitched.)

**Comment by anotherCoward** [comment]
Make sure you have an empty active inventory slot. Rapier can't be placed in backpack.

**Comment by Anon6669** [comment]
Yeah, I know. But it even happens when I have no items at all, yet.

**Comment by ntjz-kakarot** [comment]
Hello. It seems the issue is still there. My game ID 7137548122, where I played PL, I could not get the items delivered by the courier. It did for the first few minutes, but every item after I got the "Robe of the Magi" for the defusal blade, didn't come to me.
